eng/pipelines/fluent_java_mgmt_docs.yml

```diff
@@ -0,0 +1,44 @@
+trigger: none #this is not a CI, the pipeline will be triggered manually
+
+variables:
+- group: 'GitHub Release Variables'
+- name: 'GH_TOKEN'
+  value: "$(Github-azuresdkci-personalaccesstoken)"
+ 
+
+jobs:
+  - job: 'Publish_API_docs_for_the_fluent_management_java_SDK'
+  
+    steps:
+      - task: Npm@1
+        displayName: 'npm install at root'
+        inputs:
+          command: 'install'
+          
+      - script: 'chmod 775 eng/pipelines/scripts/repo'
```

Comment:
Shouldn't we just commit the script with the correct permissions instead of setting them each time in this pipeline?

Reply:
SO, the file that I am replacing is downloaded once we do "npm install".. its not natively present in the repo... cant really fix smtin that I don't control...